src/App.js

```diff
@@ -1,5 +1,26 @@
+import { Console } from "@woowacourse/mission-utils";
+
+import { parseInput } from "./inputParser.js";
+import { calculateSum } from "./calculator.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const input = await this.getInput();
+      const numbers = parseInput(input);
+      const result = calculateSum(numbers);
+      this.printResult(result);
+    } catch (error) {
+      throw new Error("[ERROR] " + error.message);
+    }
+  }
+
+  async getInput() {
+    return await Console.readLineAsync("덧셈할 문자열을 입력해 주세요.\n");
+  }
```

Comment:
```suggestion
  getInput() {
    return Console.readLineAsync("덧셈할 문자열을 입력해 주세요.\n");
  }
```
`getInput` 메서드에 추가적인 로직이 없는 것 같은데 `async/await`을 제거하는건 어떨까요?!

Reply:
오! 추가적인 로직이 없어 바로 반환되기때문에 `await`을 제거하는게 깔끔하다는 것에 동의합니다!
그런데 `getInput`을 사용할 때 이 함수가 비동기임을 명시해주기 위해서 `async`는 유지할 것 같습니다. 혹시 이 건에 대해서는 우혁님은 어떻게 생각하시나요?

Reply:
Console.readLineAsync는 Promise를 반환하는 함수이기 때문에 await를 지우면 상황에 따라 정상작동 하지 않을 수도 있습니다.

Reply:
비동기를 명시하는 점 좋은 것 같네요! 

하지만 `getInput` 메서드에 `async` 키워드를 붙인다고 해서 사용할 때 인터페이스를 봐도 알수 없고 메서드 내부를 봐야 알 수 있기 때문에 `async`를 메서드 이름에 포함한다면 사용처에서도 바로 비동기 로직인걸 확인해서 `async/await`를 사용할 수 있을 것 같네요🙃

Reply:
> Console.readLineAsync는 Promise를 반환하는 함수이기 때문에 await를 지우면 정상작동 하지 않을 수 있습니다.

넵 하지만 지금은 한번 추상화 되어있는 `getInput` 메서드이기 때문에 외부 사용처에서만 `await`를 사용해준다면 정상적으로 Promise를 다룰 수 있을 것 같아요~!

Reply:
> > Console.readLineAsync는 Promise를 반환하는 함수이기 때문에 await를 지우면 정상작동 하지 않을 수 있습니다.
> 
> 넵 하지만 지금은 한번 추상화 되어있는 `getInput` 메서드이기 때문에 외부 사용처에서만 `await`를 사용해준다면 정상적으로 Promise를 다룰 수 있을 것 같아요~!

엇 네 말씀하신 게 맞습니다. 제가 자세히 읽어보지 못했네요 ...죄송합니다

Reply:
아닙니다! 저도 남의 코드를 볼 때 제가 작성했던 코드를 기반으로 봐서 헷갈릴 때가 많더라구요🤣